WalletWasabi/Services/SingleInstanceChecker.cs

```diff
@@ -12,7 +12,7 @@ namespace WalletWasabi.Services;
 
 public class SingleInstanceChecker : BackgroundService, IAsyncDisposable
 {
-	private const string WasabiMagicString = "InCryptoWeTrust";
+	private const string WasabiMagicString = "InBitcoinWeTrust";
```

Comment:
ACK. How did that even get into the codebase?

Reply:
:D good question. I was in the DOGE coin hype, maybe.